### ISSUE TITLE
[css-env] Safe Area Insets need Logical versions

### ISSUE THREAD

**Issue by jensimmons** [w3c/csswg-drafts#6379]
Currently, the [Environment Variables specification defines four variables](https://drafts.csswg.org/css-env-1/#safe-area-insets), all physical:
```
safe-area-inset-top
safe-area-inset-right
safe-area-inset-bottom
safe-area-inset-left
```

At first glance, having physical definitions seems to be all we need. We are, after all, describing literally the top, right, bottom and left edges of a device. 

However, we do need logical versions, presumably:
```
safe-area-inset-block-start
safe-area-inset-block-end
safe-area-inset-inline-start
safe-area-inset-inline-end
```

Why? For use cases like this, where the text headline & paragraphs need an inline-start margin to push them away from the left edge. 

<img width="1179" alt="Screen Shot 2021-06-14 at 2 52 04 PM" src="https://user-images.githubusercontent.com/108474/121943942-17fc7280-cd20-11eb-884f-d0fab0163a18.png">

To support LTR + horizontal-tb, I can just write `margin-left: env(safe-area-inset-left)`, but to support multiple writing modes, I should use `margin-inline-start: env(safe-area-inset-inline-start);`

**Comment by fantasai** [comment]
@jensimmons Which writing mode are you wanting to reference? The root element or the element itself? I'm guessing the latter?

**Comment by emilio** [comment]
Yeah, that's tricky, since it means that the value of the env var depends on where it's applied to... Why is `margin-left: env(safe-area-inset-left); margin-right: env(safe-area-inset-right);` not a solution for the use case above? Is the idea that you want the image to overflow through the safe area?

**Comment by jensimmons** [comment]
> Why is margin-left: env(safe-area-inset-left); margin-right: env(safe-area-inset-right); not a solution for the use case above? 

Well, in that example, the text (h1, h2, & p) has a margin that reaches the safe area, while the photo does not — it extends out to the edge of the screen. I'm using `padding-left: env(safe-area-inset-left);` on the text, while the photo has `padding-right: 0;`. 

Like everything for logical properties, the value in having logical versions of environment variables is in providing internationalization support. Imagine this demo published in multiple languages, including RTL scripts. I want to write one set of code, not two. 

I could write:
```
.ltr h1, .ltr h2, .ltr p { 
     padding-left: env(safe-area-inset-left); 
}
.rtl h1, .rtl h2, .rtl p { 
     padding-right: env(safe-area-inset-right); 
}
```

With logical values, I could write this instead:
```
h1, h2, p { 
     padding-inline-start: env(safe-area-inset-inline-start);
}
```

And yes, I'm imagining this to work on the root writing mode, not the writing mode of the element itself.

**Comment by samhenrigold** [comment]
Has there been any movement here?
